packages/@ember/-internals/glimmer/lib/helpers/get.ts

```diff
@@ -23,33 +23,70 @@ import { CachedReference, referenceFromParts, UPDATE } from '../utils/references
 
   For example, these two usages are equivalent:
 
+  ```app/components/developer-detail.js
+  import Component from '@glimmer/component';
+
+  export default class extends Component {
+    developer = {
```

Comment:
~Same question RE: tracked~ nvm, I see the more complicated example below with tracking